src/components/weather/code/less/_weather.less

```diff
@@ -0,0 +1,35 @@
+/*------------------------------------*\
+    #Weather
+\*------------------------------------*/
+
+.component-weather {
+  width:auto;
+  text-align:center;
+  width:100px;
+  div {
+    &.cloudy {.weather-variant(cloudy);}
+    &.fog {.weather-variant(fog);}
+    &.hail {.weather-variant(hail);}
+    &.heavy-rain {.weather-variant(heavy-rain);}
+    &.heavy-snow {.weather-variant(heavy-snow);}
+    &.light-rain {.weather-variant(light-rain);}
+    &.light-snow {.weather-variant(light-snow);}
+    &.night-clear {.weather-variant(night-clear);}
+    &.night-partly-cloudy {.weather-variant(night-partly-cloudy);}
+    &.partly-cloudy {.weather-variant(partly-cloudy);}
+    &.storm {.weather-variant(storm);}
+    &.sunny {.weather-variant(sunny);}
+    &.windy {.weather-variant(windy);}
+  }
+}
+
+.weather-variant(@weather) {
+  background:url('http://brand.holidayextras.com/img/ui-toolkit/icons-weather/@{weather}.svg') no-repeat;
```

Comment:
Sorry to stick my view in at this late stage... but shouldn't we use protocol-less urls anyway? `//brand.holidayextras...`